ui/app/AppLayouts/Profile/Sections/AdvancedContainer.qml

```diff
@@ -6,16 +6,15 @@ import "../../../../imports"
 import "../../../../shared"
 import "../../../../shared/status"
 
-Item {
+ScrollView {
     id: advancedContainer
-    Layout.fillHeight: true
-    Layout.fillWidth: true
-    clip: true
-
+    anchors.top: parent.top
```

Comment:
It is a known anti-pattern to have anchors or Layout attached properties at the root Item.
The reason is that if you defined anchors, you won't be able to use this Item (AdvancedContainer) in *Layout.
If you defined Layout.* properties here and then anchors in the parent - these would conflict raising QML warnings.
A good pattern is to define anchors and/or Layout.* properties when you use this component, because it is parent's responsibility to position Items. No Item itself should be responsible for its own position.
All above is just my humble opinion.

Reply:
@pinebit no opinion on the above yet ;). I just tried to follow your suggestion, let me know if that's ok now